src/Workspaces/Core/Portable/AddImports/IAddImportsService.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#nullable enable
```

Comment:
HasExistingImport and GetImportContainer can take a null contextLocation. Possible AddImports too?